shared/templates/sysctl/oval.template

```diff
@@ -249,6 +264,12 @@
     {{{ sysctl_match() }}}
   </ind:textfilecontent54_object>
 
+  <ind:textfilecontent54_object id="object_static_lib_sysctld_{{{ rule_id }}}" version="1">
+    <ind:path>/lib/sysctl.d</ind:path>
+    <ind:filename operation="pattern match">^.*\.conf$</ind:filename>
+    {{{ sysctl_match() }}}
+  </ind:textfilecontent54_object>
+
```

Comment:
This object should not be included in the RHEL data streams.
Please, move its declaration into the conditional below.